exasol/analytics/audit/audit.py

```diff
@@ -71,27 +91,33 @@ def augment(self, queries: Iterator[Query]) -> Iterator[str]:
                 )
 
     def _insert(self, query: AuditQuery) -> str:
```

Comment:
This code has been refactored and some parts have been moved to a central method `_insert_statement()`.